src/main.cpp

```diff
@@ -189,9 +192,15 @@ int main (int argc, const char * argv[])
 	
 	vm.log_file = getenv("SAPF_LOG");
 	if (!vm.log_file) {
-		const char* home_dir = getenv("HOME");
-		char logfilename[PATH_MAX];
-		snprintf(logfilename, PATH_MAX, "%s/sapf-log.txt", home_dir);
+		#ifdef _WIN32
+			const char* home_dir = getenv("USERPROFILE");
+			char logfilename[PATH_MAX];
+			snprintf(logfilename, PATH_MAX, "%s\\sapf-log.txt", home_dir);
```

Comment:
iirc Windows accepts forward slash too, does it not work here? if it does, i think it is better to not duplicate the path join. since we are on C++17, we could potentially move the whole codebase to use the join from `std::path` later

Reply:
indeed, it works fine with forward slash!